tests/sweep_runner/test_param_sweep_runner.py

```diff
@@ -422,6 +501,55 @@ def sweep_method(data: np.ndarray, param_1: int):  # type: ignore
         "httomo.method_wrappers.generic.import_module", return_value=FakeModule
     )
 
+    preview = PreviewConfig(
+        angles=PreviewDimConfig(start=0, stop=180),
+        detector_y=PreviewDimConfig(start=50, stop=53),
+        detector_x=PreviewDimConfig(start=0, stop=160),
+    )
```

Comment:
Similar comment to another one about having extra hardcoded values than needed, could you replace these hardcoded values with the variable `GLOBAL_SHAPE` where possible, something like the following:
```suggestion
    preview = PreviewConfig(
        angles=PreviewDimConfig(start=0, stop=GLOBAL_SHAPE[0]),
        detector_y=PreviewDimConfig(start=50, stop=53),
        detector_x=PreviewDimConfig(start=0, stop=GLOBAL_SHAPE[2]),
    )
```